playbooks/common/openshift-cluster/upgrades/upgrade_nodes.yml

```diff
@@ -50,3 +50,5 @@
 - include: ../reset_excluder.yml
   tags:
   - always
+  vars:
+    target_hosts: oo_nodes_to_config
```

Comment:
Or `oo_nodes_to_upgrade` instead? If so, it will not be the same as for ``disable_excluders`` which set hosts to `oo_nodes_to_config`